services/tns_queue/tns_queue.py

```diff
@@ -41,7 +41,15 @@
 queue = []
 
 
-def tns_submission(obj_ids, tnsrobot_id, user_id, reporters="", parent_session=None):
+def tns_submission(
+    obj_ids,
+    tnsrobot_id,
+    user_id,
+    reporters="",
+    archival=False,
+    archivalComment="",
```

Comment:
let's use snake_case for the arguments, to keep it python friendly.

Reply:
yes